update-dois/src/main/java/org/reactome/release/updateDOIs/UpdateDOIs.java

```diff
@@ -43,15 +47,17 @@ public static void findAndUpdateDOIs(long authorIdTR, long authorIdGK, String pa
 		GKInstance instanceEditGK = UpdateDOIs.createInstanceEdit(UpdateDOIs.dbaGkCentral, authorIdGK, creatorFile);
 		// Gets the updated report file if it was provided for this release
 		HashMap<String, HashMap<String,String>> expectedUpdatedDOIs = UpdateDOIs.getExpectedUpdatedDOIs(pathToReport);
+		if (expectedUpdatedDOIs.size() == 0) {
+			logger.warn("No DOIs listed in UpdateDOIs.report. Please add expected DOI and displayName to UpdateDOIs.report.");
+		}
```

Comment:
If there are no expected DOIs, should the program exit or can it continue?

Reply:
I'm not sure but I think that if `expectedUpdatedDOIs` is empty, then any DOIs will end up in `notUpdated`. I don't think anything would get updated in the database, but maybe it's a good idea to just exit early if there's nothing to do.

...unless I'm wrong, and something _will_ happen, even when `expectedUpdatedDOIs` is empty.

Reply:
Nothing will happen, I'll add a return statement